jobs/azWebAppDeployJob.yml

```diff
@@ -232,6 +247,12 @@ jobs:
           steps:
           - checkout: self
 
```

Comment:
no exactly part of this PR, but the AC test steps after, there seems to be a lot of them. Would it be worth splitting them out to another template, so we can just reference 'template: azWebAppAcceptanceTests.yml' ?

Reply:
A good idea, there's a decent amount that can be refactored. However the AC tests use a decent amount of parameters that are unpacked just above, I'll create a ticket for this but don't think it's needed for this PR